use-timescale/data-tiering/querying-tiered-data.md

```diff
@@ -23,95 +21,170 @@ sessions.
 With tiered reads enabled, you can query your data normally even when it's distributed across different storage tiers.
 Your hypertable is spread across the tiers, so queries and `JOIN`s work and fetch the same data as usual.
 
-<!-- vale Google.Acronyms = YES -->
-
 <Highlight type="warning">
```

Comment:
I'd make this into a sentence without the warning and link to the performance section.  if you must, make it an info admomition.